backend/repositories/ServiceRepo.js

```diff
@@ -32,7 +32,11 @@ module.exports = class ServiceRepository {
     }
 
     async getReviews() {
```

Comment:
curious why you're using classes to store state outside of react's state management

Reply:
No state management here, this class gathers all the data for my service endpoint to keep the code clean in the controller